eng/config/PublishData.json

```diff
@@ -55,6 +55,7 @@
       "Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.CSharp": "arcade",
       "Microsoft.CodeAnalysis.ExternalAccess.AspNetCore": "vs-impl",
       "Microsoft.CodeAnalysis.ExternalAccess.Razor": "vs-impl",
+      "Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler": "vs-impl",
```

Comment:
It seems other compiler packages uses arcade. Tag @jjonescz to make sure vs-impl is the right one to use.